src/Features/Core/Portable/Completion/Providers/SymbolMatchPriority.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.CodeAnalysis.Completion.Providers
+{
+    internal class SymbolMatchPriority
+    {
+        internal static int Keyword = 100;
+        internal static int PreferType = 200;
+        internal static int PreferNamedArgument = 250;
+        internal static int PreferEventOrMethod = 300;
+        internal static int PreferFieldOrProperty = 400;
```

Comment:
Oh, and we would have to pre-select it even if the type didn't match in that case I guess.